doc/auth/saml.md

```diff
@@ -0,0 +1,133 @@
+## Overview
+
+This guide will walk through using Pachyderm's SAML support.
+These features aren't integrated into mainline Pachyderm yet and aren't
+available in any official releases. This will describe the process of:
+
+1. Activating Pachyderm enterprise and Pachyderm auth
+1. Configuring Pachyderm's auth system to enable its SAML ACS and receive SAML
+    assertions
+1. Logging in to both the dash and CLI
+1. Enabling debug logging in case anything goes wrong
+
+## Activation
+
+When starting out, we **highly** recommend running Pachyderm in Minikube, in case
+any configuration bugs make it necessary to restart the cluster.
+
+To activate Pachyderm enterprise and Pachyderm auth:
+
+```
+pachctl enterprise activate <enterprise code>
+pachctl auth activate --initial-admin=robot:admin
+```
+
+At this point, Pachyderm is ready to authenticate & authorize users.
+
+What the `--initial-admin` flag does is this:
+1. Pachyderm requires there to be at least one cluster admin if auth is
+   activated
+1. Pachyderm's authentication is built around GitHub by default. Without this
+   flag, Pachyderm asks the caller to go through an OAuth flow with GitHub, and
+   then at the conclusion, makes the caller the cluster admin. Then whoever
+   activated Pachyderm's auth system can modify it by re-authenticating via
+   GitHub and performing any necessary actions
+1. To avoid the OAuth flow, though, it's also possible to make the initial
+   cluster admin a "robot user". This is what
+   `--initial-admin=robot:<something>` does.
+1. Pachyderm will print out a Pachyderm token that authenticates the holder as
+   this robot user. At any point, you can authenticate as this robot user by
+   running
+   ```
+   $ pachctl auth use-auth-token
+   Please paste your Pachyderm auth token:
+   <paste robot token emitted by "pachctl auth activate --initial-admin=robot:admin">
+   $ # you are now robot:admin, cluster administrator
+   ```
+
+## Create IdP test app
+Here is the configuration I used for an Okta test app that authenticates Okta users
```

Comment:
It seems a little weird referencing yourself in these docs. You might want to say, Pachyderm is test with an Okta app with the following configuration.

Reply:
Done. I generally reworded this to talk about "this example" and "the example app" and "example config" and such